benefits/core/models.py

```diff
@@ -107,6 +107,10 @@ class EligibilityVerifier(models.Model):
     form_sub_placeholder = models.TextField(null=True)
     # A regular expression used to validate the 'sub' API field before sending to this verifier
     form_sub_pattern = models.TextField(null=True)
+    # Input mode can be "numeric", "tel", "search", etc. to override default "text" keyboard on mobile devices
+    form_inputmode = models.TextField(null=True)
```

Comment:
This is being pedantic, but it's weird to have one of these not use the full `snake_case` pattern. 

```suggestion
    form_input_mode = models.TextField(null=True)
```

Reply:
Yeah I thought about that too - BUT - HTML uses `inputmode`: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode so that's why I chose `inputmode`.

Reply:
But it also uses `maxlength` and that one is `form_max_length` soooo...?

Reply:
Trueeee ok